booking-modal-1.html

```diff
@@ -4,18 +4,18 @@
   <head>
     <meta charset="UTF 8" />
     <meta name="viewport" content=""width=device-width, initial-scale=1.0">
-    <title>Escape room booking modal</title>
+    <title>Escape room booking modal 1</title>
```

Comment:
Maybe you should consider renaming the title to something more like "Escape Room - Booking"?